### DISCUSSION TITLE
WooCommerce support

### DISCUSSION THREAD

**Discussion by jmotes** [WebDevStudios/nextjs-wordpress-starter#345] [Q&A]
Are there any plans to add WooCommerce support? I am just now learning to use Next.js (I have a mostly Vue.js background) but might be interested in contributing this in a couple of months if the work isn't currently planned.

**Comment by josh-tt** [comment]
Also curious about this. Before investing the time into exploring this starter, it would be good to know the 'gotchas' in terms of getting woo running in this system at the very least. What would it take to get Woo setup in such an environment (there are some nice demos online by other devs with woo/next, but all very basic) and what kind of issues can one expect to run into in doing so. I'm fairly certain it's not trivial so that's kept me from exploring this starter, but it's possible I'm mistaken.
